apps/site/pages/docs/multiple-page-templates.mdx

```diff
@@ -0,0 +1,103 @@
+Note that the product mentioned above is in the Technology `department` from the catalog. Then, you can create a `department` template.
+
+![PDP Department template](https://vtexhelp.vtexassets.com/assets/docs/src/pdpdepartment___387cf641f675d05666afeafc65a3e56b.png)
+
+If this product were under a `subcategory`, would also be possible to create a template representing this scenario. The category tree is used in the template input, separated by slashes.
+
+![PDP subcategory template](https://vtexhelp.vtexassets.com/assets/docs/src/pdpsubcategory___566be1fd3bba2db777dd18881933292c.png)
+
+You can also see the category hierarchy through the breadcrumb.
+
+![PDP breadcrumb](https://vtexhelp.vtexassets.com/assets/docs/src/breadcrumb___c9c12da51f685572dfb3a9403168b26a.png)
+
+If there are no matches for slug or `department`, the generic template will be used to render this PDP. So you need to create a document with an empty value in the template input field.
+
+![PDP general template](https://vtexhelp.vtexassets.com/assets/docs/src/pdpgeneral___cdeeaf6060b529aea029e2e5814efc50.png)
+
+Possible templates in CMS from this product:
+
+- Slug: `/apple-magic-mouse/p`
+- Department: `/technology/`
+- PDP generic template: (empty value)
+
+### PLP Template
+
+The PLP multiple-page template approach follows a similar logic to the PDP, but for PLPs FastStore will use only the slug information to find the appropriate document (template) to render. The order should be as follows:
+
+1.  A PLP document that matches the slug;
+    1. A PLP document that matches the `subcategory` slug;
+    2. A PLP document that matches the `category` slug;
+    3. A PLP document that matches the `department` (collection) slug;
+2.  If no matches are found, use the generic PLP template.
+
+Let's see the examples below:
+While using PLPs, note that this type of page is recognized by the slug pattern [https://starter.vtex.app/office/chairs](https://starter.vtex.app/office/chairs).
+
+You can create the template based on the slug: `/office/chairs`
+
+![PLP document list](https://vtexhelp.vtexassets.com/assets/docs/src/plplist___854bd00fbbcc43d8ae25169917e85632.png)
+
+Note that the slug above represents the `department` and `category` separated by slashes (department/category).
+
+The products in VTEX catalog are under a category tree, usually `/department/category/subcategory`.
+When the slug template exists, it will be rendered.
+
+There are cases where you would like to render a specific template for products inside the `department` without considering the `category` or even a `subcategory`. For this case, you can create a `department` template [/office](https://starter.vtex.app/office/chairs).
+
+![PLP department template](https://vtexhelp.vtexassets.com/assets/docs/src/plpdepartment___4c520237721ca14a722215a93e8cd4c7.png)
+
+Bear in mind that if another most specific template exists, like `/office/chairs`, the most specific will be rendered if the slug matches the same pattern.
```

Comment:
```suggestion
<Callout type="warning" emoji="⚠️">
Keep in mind that if a more specific template, such as `/office/chairs`, exists, it will take precedence when the slug matches the same pattern.
</Callout>
```